pol-core/bscript/compiler/astbuilder/ExpressionBuilder.cpp

```diff
@@ -274,6 +290,41 @@ std::vector<std::unique_ptr<Expression>> ExpressionBuilder::expressions(
   return expressions;
 }
 
+std::vector<std::unique_ptr<Expression>> ExpressionBuilder::expressions(
+    std::vector<EscriptGrammar::EscriptParser::InterpolatedStringPartContext*> ctx )
```

Comment:
This is the function that takes the Parse Tree node `InterpolatedStringPartContext` and returns the list of its children expressions, ie. `vector<Expression>`